browser/profiles/profile_util.cc

```diff
@@ -215,6 +220,15 @@ bool IsRegularProfile(content::BrowserContext* context) {
          profile->IsRegularProfile();
 }
 
+void RecordInitialP3AValues(Profile* profile) {
+  bool is_sponsored_image_enabled =
+      profile->GetPrefs()->GetBoolean(kNewTabPageShowBackgroundImage) &&
```

Comment:
These two value can be changed during the runtime.
We don't need to track this also?

Reply:
Not sure what you mean? I put here so that it would set an initial value on launch - instead of capturing if someone changes it live (which is less likely to happen).

Reply:
ah - I understand 😄 Since this is reported only once a week, I think that during launch is good enough. But I can also track if the value changes. WDYT, @iefremov

Reply:
Do we track how often users restart? I would imagine most only restart when there's an update, so every 3 - 6 weeks. If we want accurate then we should probably report the state at the time the data is sent to p3a.

Reply:
maybe we can ask the metric owner for their preference - both ways could work. 
I'm thinking of adding some more user-friendly P3A APIs  for cases like this, e.g. picking some values at transmission time, as @petemill suggests.

Reply:
Updated - I capture all places it changes and update P3A accordingly 👍